hikari/audit_logs.py

```diff
@@ -216,6 +220,72 @@ class MessagePinEntryInfo(BaseAuditLogEntryInfo):
     message_id: snowflakes.Snowflake = attr.ib(repr=True)
     """The ID of the message that's being pinned or unpinned."""
 
+    async def fetch_channel(self) -> channels.TextChannel:
+        """Fetch the object of the channel where a pinned message is being targeted.
+
+        Returns
+        -------
+        hikari.channels.TextChannel
+            The channel where a pinned message is being targeted.
```

Comment:
Better wording for this would be: `The channel where this message was pinned or unpinned.`